package.json

```diff
@@ -13,7 +13,6 @@
     "url": "git+https://github.com/Microsoft/fast-dna.git"
   },
   "scripts": {
```

Comment:
Are there any other scripts? Can we remove the entire reference to scripts?

Reply:
Yes. I have another PR up now adds three new scripts for Integration-testing.